source/Mission.cpp

```diff
@@ -177,6 +179,28 @@ void Mission::Load(const DataNode &node)
 						" They are now mission-level properties:");
 			}
 		}
+		else if(child.Token(0) == "outfit" && child.Size() >= 3)
+		{
+			if(!outfitStr.empty())
+				node.PrintTrace("Warning: more than one outfit objective, only last one will be used:");
+			outfitStr = child.Token(1);
+			outfitUnits = child.Value(2);
+			if(child.Size() >= 4)
+				outfitLimit = child.Value(3);
+			if(child.Size() >= 5)
+				outfitProb = child.Value(4);
+		}
+		else if(child.Token(0) == "outfitter" && child.Size() >= 3)
+		{
+			if(!outfitterStr.empty())
+				node.PrintTrace("Warning: more than one outfitter objective, only last one will be used:");
+			outfitterStr = child.Token(1);
```

Comment:
I think you should rename this one and the other one to more explicit variables, it being a string is easy to see. Maybe outfitterName and outfitName? Or requestedOutfit and requestedOutfitter (I prefer that one) or yet another one

Reply:
Like the "requested" prefix. Making change